srcpkgs/docker-cli/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'docker-cli'
+pkgname=docker-cli
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/cli"
+hostmakedepends="pkg-config tar"
+depends="containerd runc iptables xz git"
+short_desc="Pack, ship and run any application as a lightweight container - cli"
```

Comment:
`short_desc="Command-line interface to the Docker container runtime"` captures the upstream description from the GitHub project page (which is just "The Docker CLI") with more detail but avoids the marketing speak.

Reply:
I've revised this to `short_desc="Command-line interface to the Docker daemon"` as I feel as though it's less wordy and more accurate.